masterlist.yaml

```diff
@@ -2204,6 +2204,17 @@ plugins:
     clean:
       - crc: 0x7E701274
         util: 'FNVEdit v4.0.3g'
+  - name: 'S6S Perks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/73438/'
+        name: 'Sweet 6 Shooter Perks - TTW - NV'
+    clean:
+# NV Version 16.13
+      - crc: 0xC95D69CA
+        util: 'FNVEdit v4.0.4'
+# TTW Version 16.13
+      - crc: 0x8EA841DD
+        util: 'FNVEdit v4.0.4'
```

Comment:
I will add these in the commit mesage.

# NV Version 16.13, 0xC95D69CA.
# TTW Version 16.13, 0x8EA841DD

```suggestion
    clean:
      - crc: 0xC95D69CA
        util: 'FNVEdit v4.0.4'
      - crc: 0x8EA841DD
        util: 'FNVEdit v4.0.4'
```